src/parsec_service/key_id_managers.md

```diff
@@ -0,0 +1,10 @@
+# Key ID Managers
+
+# On-Disk Key ID Manager
+
+The on-disk key ID manager stores the mapping between key triple and key ID directly on disk, in a
+folder which path is configurable.
```

Comment:
> folder with a configurable path.

or
> folder whose path is configurable.

Reply:
Will change 😃 
https://youtu.be/01Dn53H2ZLw?t=70

Reply:
don't hate the player, hate the game.